src/test/java/com/fryrank/controller/ReviewControllerTests.java

```diff
@@ -100,4 +103,106 @@ public void testAddNewReviewNullBody() throws Exception {
+            {
+                add("ChIJl8BSSgfsj4ARi9qijghUAH0");
+            }
+        };
+        final AggregateReviewFilter aggregateReviewFilter = new AggregateReviewFilter(true);
+
+        when(reviewDAL.getAggregateReviewInformationForRestaurants(restaurantIds, aggregateReviewFilter)).thenReturn(expectedOutput);
+
+        final GetAggregateReviewInformationOutput actualOutput = controller.getAggregateReviewInformationForRestaurants("ChIJl8BSSgfsj4ARi9qijghUAH0", true);
+        assertEquals(actualOutput, expectedOutput);
+    }
+
+    @Test
+    public void testGetMultipleRestaurantAllAggregateInformation() throws Exception {
+        final Map<String, AggregateReviewInformation> expectedAggregateReviewInformation = Map.of(
+                "ChIJl8BSSgfsj4ARi9qijghUAH0", new AggregateReviewInformation("ChIJl8BSSgfsj4ARi9qijghUAH0", 5.0F),
+                "ChIJ1wHcROHNj4ARmNwmP2PcUWw", new AggregateReviewInformation("ChIJ1wHcROHNj4ARmNwmP2PcUWw", 7.0F)
+        );
+        final GetAggregateReviewInformationOutput expectedOutput = new GetAggregateReviewInformationOutput(expectedAggregateReviewInformation);
+        final List<String> restaurantIds = new ArrayList<String>(){
+            {
+                add("ChIJl8BSSgfsj4ARi9qijghUAH0");
+                add("ChIJ1wHcROHNj4ARmNwmP2PcUWw");
+            }
+        };
+        final AggregateReviewFilter aggregateReviewFilter = new AggregateReviewFilter(true);
+
+        when(reviewDAL.getAggregateReviewInformationForRestaurants(restaurantIds.stream().sorted().collect(Collectors.toList()), aggregateReviewFilter)).thenReturn(expectedOutput);
+
+        final GetAggregateReviewInformationOutput actualOutput = controller.getAggregateReviewInformationForRestaurants("ChIJl8BSSgfsj4ARi9qijghUAH0,ChIJ1wHcROHNj4ARmNwmP2PcUWw", true);
+        assertEquals(actualOutput, expectedOutput);
+    }
+
+    @Test
+    public void testGetMultipleRestaurantAllAggregateInformationReverseOrdering() throws Exception {
+        final Map<String, AggregateReviewInformation> expectedAggregateReviewInformation = Map.of(
+                "ChIJl8BSSgfsj4ARi9qijghUAH0", new AggregateReviewInformation("ChIJl8BSSgfsj4ARi9qijghUAH0", 5.0F),
+                "ChIJ1wHcROHNj4ARmNwmP2PcUWw", new AggregateReviewInformation("ChIJ1wHcROHNj4ARmNwmP2PcUWw", 7.0F)
+        );
+        final GetAggregateReviewInformationOutput expectedOutput = new GetAggregateReviewInformationOutput(expectedAggregateReviewInformation);
+        final List<String> restaurantIds = new ArrayList<String>(){
+            {
+                add("ChIJl8BSSgfsj4ARi9qijghUAH0");
+                add("ChIJ1wHcROHNj4ARmNwmP2PcUWw");
+            }
+        };
+        final AggregateReviewFilter aggregateReviewFilter = new AggregateReviewFilter(true);
+
+        when(reviewDAL.getAggregateReviewInformationForRestaurants(restaurantIds.stream().sorted().collect(Collectors.toList()), aggregateReviewFilter)).thenReturn(expectedOutput);
```

Comment:
Can we look into how to reuse the code (setup method) since it's duplicated twice, if possible?